brave/src/test/java/brave/baggage/CorrelationScopeDecoratorTest.java

```diff
@@ -59,12 +59,13 @@
     .add(SingleBaggageField.remote(FLUSH_FIELD.baggageField()))
     .build();
 
-  TraceContext context = baggageFactory.decorate(TraceContext.newBuilder()
+  TraceContext context = TraceContext.newBuilder()
```

Comment:
this change caused a lot of diff, but it improves the quality. the test is the same though